test/plugins/templates.test.js

```diff
@@ -387,6 +389,56 @@ describe('template plugin test', () => {
         });
     });
 
+    describe('GET /templates/name/metrics', () => {
+        let options;
+
+        beforeEach(() => {
+            options = {
+                method: 'GET',
+                url: '/templates/screwdriver%2Fbuild/metrics'
+            };
+        });
+
+        it('returns 200 and all template versions and metrics for a template name', () => {
+            templateFactoryMock.listWithMetrics.resolves(testtemplateversionsmetrics);
```

Comment:
it's better to add this line `assert.expect(x);` with `x` being number of asserts in your test

Reply:
assert.expect() does not exist in this repo